src/esp/metadata/managers/AssetAttributesManager.h

```diff
@@ -401,7 +392,7 @@ class AssetAttributesManager
    * @param templateID the ID of the template to remove
    * @param templateHandle the string key of the attributes desired.
    */
-  void updateTemplateHandleLists(
+  void updateObjectHandleLists(
       CORRADE_UNUSED int templateID,
       CORRADE_UNUSED const std::string& templateHandle) override {}
```

Comment:
Not really caused by this PR, but I just discovered the macro isn't [ignored by Doxygen](https://aihabitat.org/docs/habitat-sim/classesp_1_1metadata_1_1managers_1_1StageAttributesManager.html#a296a2f5b66f518651286421a4bf0c829) as it should be -- can you add `CORRADE_UNUSED=` (yes, with equals at the end, to make it expand to empty) to `PREDEFINED` in [docs/Doxyfile](https://github.com/facebookresearch/habitat-sim/blob/89731b73b59251e25ef73569dc0b334629bc9bdd/docs/Doxyfile#L2136)? There's probably more of those to add, but I can't think of any other right now.

Reply:
Is there a comment I can add to it? for the future! :)